README.md

```diff
@@ -157,3 +157,10 @@ Add the configurations to the `custom.yml`. A sample is given below.
 ```
 
 Follow the steps mentioned under `docs` directory to customize/create new Ansible scripts and deploy the recommended patterns.
+
+## Performance Tuning
+
+System configurations can be changed through Puppet to optimize OS level performance. Performance tuning can be enabled by changing `enable_performance_tuning` in `dev/group_vars/apim.yml` to `true`.
```

Comment:
`System configurations can be changed through Ansible`

Reply:
Fixed https://github.com/wso2/ansible-apim/pull/45/commits/48626889a1dd269050e9be7a121221d2944181d1